modules/core/src/camera/vpCameraParameters.cpp

```diff
@@ -118,13 +118,31 @@ vpCameraParameters::vpCameraParameters(double cam_px, double cam_py, double cam_
 vpCameraParameters::vpCameraParameters(double cam_px, double cam_py, double cam_u0,
                                        double cam_v0, double cam_kud, double cam_kdu)
   : px(DEFAULT_PX_PARAMETER), py(DEFAULT_PY_PARAMETER), u0(DEFAULT_U0_PARAMETER), v0(DEFAULT_V0_PARAMETER),
-    kud(DEFAULT_KUD_PARAMETER), kdu(DEFAULT_KDU_PARAMETER), width(0), height(0), isFov(false), m_hFovAngle(0),
-    m_vFovAngle(0), fovNormals(), inv_px(1. / DEFAULT_PX_PARAMETER), inv_py(1. / DEFAULT_PY_PARAMETER),
-    projModel(DEFAULT_PROJ_TYPE)
+    kud(DEFAULT_KUD_PARAMETER), kdu(DEFAULT_KDU_PARAMETER), m_isKannalaBrandt(false), m_dist_coefs(), width(0), height(0),
+    isFov(false), m_hFovAngle(0), m_vFovAngle(0), fovNormals(), inv_px(1. / DEFAULT_PX_PARAMETER),
+    inv_py(1. / DEFAULT_PY_PARAMETER), projModel(DEFAULT_PROJ_TYPE)
 {
   initPersProjWithDistortion(cam_px, cam_py, cam_u0, cam_v0, cam_kud, cam_kdu);
 }
 
+/*!
+  Constructor for projection with Kannala-Brandt distortion model
+
+  \param cam_px,cam_py : pixel size
+  \param cam_u0,cam_v0 : principal points
+  \param coefficients  : distorsion model coefficients
+
+ */
+vpCameraParameters::vpCameraParameters(double cam_px, double cam_py, double cam_u0,
+                                       double cam_v0, std::vector<double> coefficients)
```

Comment:
`coefficients` should be passed as a const reference `const std::vector<double> &coefficients`